### DISCUSSION TITLE
Documentation source?

### DISCUSSION THREAD

**Discussion by TonyGravagno** [WordPress/Documentation-Issue-Tracker#256] [Q&A]
Where exactly is the source code/text for the documentation (developer.wordpress.org), Codex, etc?

Can we fork that repo to work on docs-as-code for a PR back to the origin?

TY

**Comment by flexseth** [comment]
Hey @TonyGravagno - know it's been a long time since you opened this discussion, but I recently started getting involved working on the Developer Documentation (AKA DevHub) - at [developer.wordpress.org](developer.wordpress.org). 

There are a couple of places you can submit PRs to DevHub, but only for two sections:
- Plugins Handbook - [developer.wordpress.org/plugins](developer.wordpress.org/plugins)
- Advanced Administration Handbook - [https://developer.wordpress.org/advanced-administration/](https://developer.wordpress.org/advanced-administration/)

In these two instances you can edit the Handbooks directly, via Pull Requests (PRs.)

## Updating code samples
If you see sample code that needs updating, that code may exist in the [Gutenberg](https://github.com/WordPress/Gutenberg) repository. I was able to find code and update it in this way by searching that repo for what I was looking for. 

- https://github.com/WordPress/gutenberg/pull/59517

These changes were directly reflected once merged into the Gutenberg repo, and updated on developer.wordpress.org. 
 
## Updating other documentation in the wild
For other documentation, it's best to open an issue here and use the prompts on the issue to describe what docs need to change. Providing updated verbiage, screenshots, etc is very helpful for the documentation authors. 

Please submit documentation update requests to the [Documentation Issue Tracker](https://github.com/WordPress/Documentation-Issue-Tracker/issues) issues in this repo! And the docs team will help to label the docs in question accurately, so they can get to the right team.

## Updating other various documentation
For the rest of the documentation, it will go through the processes of the various teams. For example, there are Theme Handbook docs teams, the end-user docs (AKA HelpHub), and more. And it really depends on where the documentation lives.

## Updating make.wordpress.org infomration
There are some Make Handbooks synced with WP. 

These are typically geared towards contributing to WordPress...

- https://github.com/WordPress/hosting-handbook
- https://github.com/WordPress/plugins-handbook
- https://github.com/WordPress/test-handbook
- https://github.com/WordPress/contributor-day-handbook

## Updating DevHub
**and some "developer" handbooks**
- https://github.com/WordPress/Advanced-administration-handbook
- https://github.com/WordPress/developer-plugins-handbook

There was recent discussion on a [Trac ticket](https://meta.trac.wordpress.org/ticket/7520) about moving more information to GitHub so it could be worked on by the community. But it appears that many teams have existing processes in place for editing different forward-facing aspects of the docs to establish guard rails for editing. 

I hope that's helpful and not too late of information. 
Or helps someone else looking for this information down the road. :)

If this changes down the road I'll update this discussion with more info!

Hopefully that helps to answer your question! Best/Seth

**Comment by TonyGravagno** [comment]
Hey Seth - Thanks for the notes! I'll go through the links as time permits to create a picture in my head for where this stuff lives, and what happens for docs that are protected within the guardrails.
